### ISSUE TITLE
Node is stuck

### ISSUE THREAD

**Issue by Insoleet** [duniter/duniter#415]
My node is stuck, on its own block. The logs show this error repeated a lot. Actually, the error was repeated so often that all my log files only contain these lines...

```
2016-05-06T19:01:26+02:00 - warn: Waiting 0s before starting computing next block...
2016-05-06T19:01:26+02:00 - error:  TypeError: Cannot read property 'powMin' of undefined
    at /opt/testnet/.duniter/app/lib/rules/global_rules.js:740:24
    at next (native)
    at onFulfilled (/opt/testnet/.duniter/node_modules/co/index.js:65:19)
    at process._tickCallback (node.js:412:9)
```

**Comment by c-geek** [comment]
This is most probably a duplicate of #431.
